CHANGELOG.md

```diff
@@ -53,6 +53,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 ### New Features and Optimizations
 - Implement Kahneman-Tversky Optimization (KTO).
 - Sequence packing is now supported when running SFT with SFTChatDataset.
+- Implemented the self rewarding and meta self rewarding algorithms.
```

Comment:
fix puncdtuation.

Implemented the self-rewarding and meta self-rewarding algorithms.